engine/app/components/citizens_advice_components/radio_group.rb

```diff
@@ -4,13 +4,9 @@ module CitizensAdviceComponents
   class RadioGroup < FormGroup
     renders_many :inputs, Checkable::Radio
 
-    def initialize(legend:, name:, id: nil, options: nil)
-      super
-      set_options(options)
-    end
```

Comment:
This isn't needed

Reply:
For my personal understanding - is it because it's inheriting from `FormGroup`?